client/src/App.js

```diff
@@ -1,18 +1,21 @@
 import React, { useState } from "react";
-import { BrowserRouter as Router, Route } from "react-router-dom";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 
 import Login from "./components/Login";
 import "./styles.scss";
+import PrivateRoute from "./components/PrivateRoute";
+import BubblePage from './components/BubblePage'
 
 function App() {
   return (
     <Router>
       <div className="App">
-        <Route exact path="/" component={Login} />
-        {/* 
-          Build a PrivateRoute component that will 
-          display BubblePage when you're authenticated 
-        */}
+        <Switch>
+          <Route exact path="/" component={Login} />
+          <PrivateRoute path='/dashboard' component={BubblePage} />
+          <Route path='/login' component={Login} />
+          <Route component={Login} />
```

Comment:
Looks like we've got three `<Route/>` components linked to the Login page. We only need one `<Route/>`; the others can be removed.